web/src/components/query-summary/stats-query-summary.tsx

```diff
@@ -1,32 +1,55 @@
 import { FlexItem, Text, TextVariants, Tooltip } from '@patternfly/react-core';
-import { GlobeAmericasIcon } from '@patternfly/react-icons';
+import { ExclamationTriangleIcon, GlobeAmericasIcon } from '@patternfly/react-icons';
 import * as React from 'react';
 import { useTranslation } from 'react-i18next';
 import './query-summary.css';
+import { formatDuration } from '../../utils/duration';
 
 export const StatsQuerySummary: React.FC<{
-  lastRefresh: Date;
+  detailed?: boolean;
+  loading?: boolean;
+  lastRefresh?: Date;
+  lastDuration?: number;
+  warningMessage?: string;
+  slownessReason?: string;
   numQueries?: number;
-}> = ({ numQueries, lastRefresh }) => {
+}> = ({ detailed, numQueries, lastRefresh, lastDuration, loading, warningMessage, slownessReason }) => {
   const { t } = useTranslation('plugin__netobserv-plugin');
 
+  const dateText = lastRefresh ? lastRefresh.toLocaleTimeString() : t('Loading...');
+
   return (
     <FlexItem>
       <Tooltip
         content={
-          <Text component={TextVariants.p}>
-            {t('Last refresh: {{time}}', {
-              time: lastRefresh.toLocaleTimeString()
-            })}
-            {numQueries && ` ${t('running')} ${numQueries} ${numQueries > 1 ? t('queries') : t('query')}`}
-          </Text>
+          <>
+            <Text>
+              {lastRefresh
+                ? t('Last refresh: {{time}}', {
+                    time: dateText
+                  })
+                : dateText}
+            </Text>
+            {numQueries && <Text>{t('Query count: {{count}}', { count: numQueries })}</Text>}
+            {lastDuration && lastDuration > 0 && (
+              <Text>{t('Duration: {{duration}}', { duration: formatDuration(lastDuration) })}</Text>
+            )}
+            {warningMessage !== undefined && (
+              <>
+                <br />
+                <Text>{warningMessage}</Text>
+                <Text>{slownessReason}</Text>
+              </>
+            )}
+          </>
         }
       >
-        <div className="stats-query-summary-container-with-icon">
-          <GlobeAmericasIcon />
+        <div className={`stats-query-summary-container-with-icon ${loading ? 'stats-loading-blink' : ''}`}>
+          {warningMessage !== undefined ? <ExclamationTriangleIcon /> : <GlobeAmericasIcon />}
           <Text id="lastRefresh" component={TextVariants.p}>
-            {lastRefresh.toLocaleTimeString()}
-            {numQueries && ` ${t('running')} ${numQueries} ${numQueries > 1 ? t('queries') : t('query')}`}
+            {dateText}
+            {detailed && numQueries && ` ${t('running')} ${numQueries} ${numQueries > 1 ? t('queries') : t('query')}`}
+            {detailed && lastDuration && lastDuration > 0 && ` ${t('in')} ${formatDuration(lastDuration)}`}
```

Comment:
not sure exactly what's wrong here, but often I end up seeing truncated text: 
![image](https://github.com/netobserv/network-observability-console-plugin/assets/2153442/277183ca-61a4-4d34-b4fe-3c334bd1cd97)

Reply:
that one should do the trick
https://github.com/netobserv/network-observability-console-plugin/pull/438/commits/64491951065e01ad723c70e9413af8ca8d54f37c

Reply:
I was just looking at `formatDuration` ... in fact if we use `formatDurationAboveMillisecond` instead we cover more durations, as formatDuration ignores when it's below a second and the other is more precise

Reply:
@jpinsonneau I've marked lgtm but still I think it would be better to use `formatDurationAboveMillisecond`, don't you think?

Reply:
sure, done in https://github.com/netobserv/network-observability-console-plugin/pull/438/commits/2e9e9ccc0c2538916ea389113c509528341db47c
Thanks !